.github/workflows/check.yaml

```diff
@@ -15,8 +15,6 @@ jobs:
       matrix:
         os:
           - ubuntu-latest
-          - macos-latest
```

Comment:
We should remove this line too now https://github.com/pyvenvmanage/PyVenvManage/blob/main/.github/workflows/check.yaml#L38